### PR TITLE
Story/quad 113

### PR DESCRIPTION
**Pull request by PawanMahalle**
Resolved conflicts and added user authentication for story/QUAD-113 as per review comments.

### CODE REVIEW

**Comment by PawanMahalle** [comment]
File changes:
[Updated] src\main\java\edu\asu\spring\quadriga\web\publicwebsite\PublicBlogController.java
[Updated] src\main\webapp\WEB-INF\views\projectblog.jsp
[Added] src\main\webapp\WEB-INF\views\addprojectblog.jsp

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by PawanMahalle** [comment]
@jdamerow  Resolved merged conflicts. Please review.
File changes:
[Updated] src\main\java\edu\asu\spring\quadriga\web\publicwebsite\PublicBlogController.java
[Updated] src\main\webapp\WEB-INF\views\projectblog.jsp
[Added] src\main\webapp\WEB-INF\views\addprojectblog.jsp
[Updated] src\main\webapp\WEB-INF\views\public\navbar.jsp

**Comment by jdamerow** [comment]
@PawanMahalle please resolve conflicts

**Comment by PawanMahalle** [comment]
Resolved conflicts. Please review.

**Comment by jdamerow** [comment]
hm, so I figured out why you changed over 1000 files. It's the line endings. Your files use LF, while the ones in develop seem to use CRLF. Or actually, kavinya changed them and you merged her changes into this branch. The thing is that is generally correct to use LF rather than CRLF since we have mixed platforms. If we change this now though, it might cause issues for everybody else. I'll think about this.

File: Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <li ${currentPage == "home" ? "class=\"active\"" : ""}><a
         href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
 <li ${currentPage == "networks" ? "class=\"active\"" : ""}><a
+	href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/projectblog">Project
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp:4]: you are still using the same currentPage variable value as networks.

Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp:4]: never mind, just saw that you changed that later on.

**Comment by jdamerow** [comment]
@PawanMahalle, is this still relevant? I said this would be in a different PR now, right? If so, please close this PR.

**Comment by PawanMahalle** [comment]
The changes in this pull request are made into consolidated PR for story/QUAD-137. Closing the request.

**Pull request closed without merging**